apps/web-giddh/src/assets/css/style-1.scss

```diff
@@ -470,9 +470,6 @@ aside-menu-account .container-fluid {
     row-gap: 20px;
 }
 
-.border-bottom {
```

Comment:
why removed this css? is this not getting used anywhere in giddh?

Reply:
this css also write in style-2.scss file